.travis.yml

```diff
@@ -46,6 +46,7 @@ install:
   # Customise the testing environment
   # ---------------------------------
   - conda config --add channels conda-forge
+  - conda config --add channels conda-forge/label/testing
```

Comment:
See https://github.com/conda-forge/matplotlib-feedstock/issues/110

Reply:
I see.  That all makes a bit more sense now.  Well This looks good to me, I guess we are just waiting for mpl 3.0.1.  

Thanks for this @QuLogic, ping me when this can be merged and I'll press the button.